### ISSUE TITLE
the process is Killed

### ISSUE THREAD

**Issue by J-T-kullo** [SCLBD/DeepfakeBench#100]
I run :
python training/train.py --detector_path ./training/config/detector/xception.yaml

when the 1 epoch's train and test finished, a new tqdm start, but it is not the 2th epoch and when it reach to 100%, the process is killed, please help me.
![image](https://github.com/user-attachments/assets/e90b6604-9831-4f57-a0b6-b97429d65a5f)

the train.log:
2024-08-20 02:36:10,550 - INFO - Save log to ./logs/training/xception_None_2024-08-20-02-36-10
2024-08-20 02:36:10,550 - INFO - --------------- Configuration ---------------
2024-08-20 02:36:10,550 - INFO - Parameters: 
log_dir: ./logs/training/
pretrained: /root/data1/DeepfakeBench-main/training/pretrained/xception-b5690688.pth
model_name: xception
backbone_name: xception
backbone_config: {'mode': 'original', 'num_classes': 2, 'inc': 3, 'dropout': False}
all_dataset: ['FaceForensics++', 'FF-F2F', 'FF-DF', 'FF-FS', 'FF-NT', 'FaceShifter', 'DeepFakeDetection', 'Celeb-DF-v1', 'Celeb-DF-v2', 'DFDCP', 'DFDC', 'DeeperForensics-1.0', 'UADFV']
train_dataset: ['FaceForensics++']
test_dataset: ['FaceForensics++', 'DeepFakeDetection']
compression: c23
train_batchSize: 32
test_batchSize: 32
workers: 8
frame_num: {'train': 32, 'test': 32}
resolution: 256
with_mask: False
with_landmark: False
use_data_augmentation: True
data_aug: {'flip_prob': 0.5, 'rotate_prob': 0.0, 'rotate_limit': [-10, 10], 'blur_prob': 0.5, 'blur_limit': [3, 7], 'brightness_prob': 0.5, 'brightness_limit': [-0.1, 0.1], 'contrast_limit': [-0.1, 0.1], 'quality_lower': 40, 'quality_upper': 100}
mean: [0.5, 0.5, 0.5]
std: [0.5, 0.5, 0.5]
optimizer: {'type': 'adam', 'adam': {'lr': 0.0002, 'beta1': 0.9, 'beta2': 0.999, 'eps': 1e-08, 'weight_decay': 0.0005, 'amsgrad': False}, 'sgd': {'lr': 0.0002, 'momentum': 0.9, 'weight_decay': 0.0005}}
lr_scheduler: None
nEpochs: 10
start_epoch: 0
save_epoch: 1
rec_iter: 100
logdir: ./logs
manualSeed: 1024
save_ckpt: True
save_feat: True
loss_func: cross_entropy
losstype: None
metric_scoring: auc
cuda: True
cudnn: True
mode: train
lmdb: True
dry_run: False
rgb_dir: ./datasets/rgb
lmdb_dir: ./datasets/lmdb
dataset_json_folder: preprocessing/dataset_json
SWA: False
save_avg: True
task_target: None
label_dict: {'DFD_fake': 1, 'DFD_real': 0, 'FF-SH': 1, 'FF-F2F': 1, 'FF-DF': 1, 'FF-FS': 1, 'FF-NT': 1, 'FF-FH': 1, 'FF-real': 0, 'CelebDFv1_real': 0, 'CelebDFv1_fake': 1, 'CelebDFv2_real': 0, 'CelebDFv2_fake': 1, 'DFDCP_Real': 0, 'DFDCP_FakeA': 1, 'DFDCP_FakeB': 1, 'DFDC_Fake': 1, 'DFDC_Real': 0, 'DF_fake': 1, 'DF_real': 0, 'UADFV_Fake': 1, 'UADFV_Real': 0, 'roop_Real': 0, 'roop_Fake': 1}
local_rank: 0
ddp: False

2024-08-20 02:36:14,182 - INFO - Load pretrained model successfully!
2024-08-20 02:36:15,925 - INFO - ===> Epoch[0] start!
2024-08-20 02:36:16,050 - INFO - data_dict saved to ./logs/training/xception_None_2024-08-20-02-36-08/train/FaceForensics++/data_dict_train.pickle
2024-08-20 02:37:21,689 - INFO - Iter: 0    training-loss, overall: 0.7332237958908081    training-loss, cls: 0.7332237958908081    
2024-08-20 02:37:21,839 - INFO - Iter: 0    training-metric, acc: 0.28125    training-metric, auc: 0.5703703703703704    training-metric, eer: 0.4    training-metric, ap: 0.8859899089209624    
2024-08-20 02:39:02,951 - INFO - Iter: 300    training-loss, overall: 0.417162150144577    training-loss, cls: 0.417162150144577    
2024-08-20 02:39:02,952 - INFO - Iter: 300    training-metric, acc: 0.8207291666666666    training-metric, auc: 0.7739294910649579    training-metric, eer: 0.2915074555074559    training-metric, ap: 0.9294036690115086    
2024-08-20 02:40:07,680 - INFO - Iter: 600    training-loss, overall: 0.32740318775177    training-loss, cls: 0.32740318775177    
2024-08-20 02:40:07,681 - INFO - Iter: 600    training-metric, acc: 0.8567708333333334    training-metric, auc: 0.8919917724286751    training-metric, eer: 0.17610733710733717    training-metric, ap: 0.9694849261296061    
2024-08-20 02:41:13,999 - INFO - Iter: 900    training-loss, overall: 0.2866016924381256    training-loss, cls: 0.2866016924381256    
2024-08-20 02:41:13,999 - INFO - Iter: 900    training-metric, acc: 0.8767708333333334    training-metric, auc: 0.9211456915079659    training-metric, eer: 0.14625744625744636    training-metric, ap: 0.9783137953736278    
2024-08-20 02:42:17,705 - INFO - Iter: 1200    training-loss, overall: 0.26209184527397156    training-loss, cls: 0.26209184527397156    
2024-08-20 02:42:17,706 - INFO - Iter: 1200    training-metric, acc: 0.8910416666666666    training-metric, auc: 0.9359697821559287    training-metric, eer: 0.12821588596588604    training-metric, ap: 0.9827393037773332    
2024-08-20 02:43:19,058 - INFO - Iter: 1500    training-loss, overall: 0.25036290287971497    training-loss, cls: 0.25036290287971497    
2024-08-20 02:43:19,058 - INFO - Iter: 1500    training-metric, acc: 0.8958333333333334    training-metric, auc: 0.945393020047359    training-metric, eer: 0.10630553021857364    training-metric, ap: 0.984894868465333    
2024-08-20 02:44:24,787 - INFO - Iter: 1800    training-loss, overall: 0.2331622838973999    training-loss, cls: 0.2331622838973999    
2024-08-20 02:44:24,788 - INFO - Iter: 1800    training-metric, acc: 0.90125    training-metric, auc: 0.9524683964435843    training-metric, eer: 0.09728971953971953    training-metric, ap: 0.9870150650393068    
2024-08-20 02:45:29,615 - INFO - Iter: 2100    training-loss, overall: 0.21420705318450928    training-loss, cls: 0.21420705318450928    
2024-08-20 02:45:29,615 - INFO - Iter: 2100    training-metric, acc: 0.9122916666666666    training-metric, auc: 0.9580900944408374    training-metric, eer: 0.09710918710918709    training-metric, ap: 0.9895019374295813    
2024-08-20 02:46:34,407 - INFO - Iter: 2400    training-loss, overall: 0.21074503660202026    training-loss, cls: 0.21074503660202026    
2024-08-20 02:46:34,408 - INFO - Iter: 2400    training-metric, acc: 0.9110416666666666    training-metric, auc: 0.964595867790006    training-metric, eer: 0.07947594072594069    training-metric, ap: 0.9904233271998774    
2024-08-20 02:47:39,369 - INFO - Iter: 2700    training-loss, overall: 0.21064816415309906    training-loss, cls: 0.21064816415309906    
2024-08-20 02:47:39,370 - INFO - Iter: 2700    training-metric, acc: 0.913125    training-metric, auc: 0.9620334505250305    training-metric, eer: 0.08898772523772515    training-metric, ap: 0.9894058881404656    
2024-08-20 02:48:39,295 - INFO - Iter: 3000    training-loss, overall: 0.20685704052448273    training-loss, cls: 0.20685704052448273    
2024-08-20 02:48:39,295 - INFO - Iter: 3000    training-metric, acc: 0.9133333333333333    training-metric, auc: 0.9647736928226459    training-metric, eer: 0.07595798645798642    training-metric, ap: 0.9901585202207036    
2024-08-20 02:49:35,697 - INFO - Iter: 3300    training-loss, overall: 0.199499249458313    training-loss, cls: 0.199499249458313    
2024-08-20 02:49:35,698 - INFO - Iter: 3300    training-metric, acc: 0.91625    training-metric, auc: 0.9672262747123018    training-metric, eer: 0.07430062530062531    training-metric, ap: 0.9916937568761028    
2024-08-20 02:50:32,214 - INFO - ===> Test start!
2024-08-20 02:50:32,263 - INFO - data_dict saved to ./logs/training/xception_None_2024-08-20-02-36-08/test/FaceForensics++/data_dict_test.pickle
2024-08-20 02:51:18,850 - INFO - Metrics saved to ./logs/training/xception_None_2024-08-20-02-36-08/test/FaceForensics++/metric_dict_best.pickle
2024-08-20 02:51:18,863 - INFO - dataset: FaceForensics++    step: 3590    testing-loss, overall: 0.1814987063407898    testing-loss, cls: 0.1814987063407898    
2024-08-20 02:51:18,890 - INFO - dataset: FaceForensics++    step: 3590    testing-metric, acc: 0.927461139896373    testing-metric, auc: 0.972033441721588    testing-metric, eer: 0.08126814020986828    testing-metric, ap: 0.9930476336225846    testing-metric, video_auc: 0.9979591836734694    testing-metric, acc_real:0.9278856887698147; acc_fake:0.9273549611926964
2024-08-20 02:51:19,101 - INFO - data_dict saved to ./logs/training/xception_None_2024-08-20-02-36-08/test/DeepFakeDetection/data_dict_test.pickle

**Comment by J-T-kullo** [comment]
![image](https://github.com/user-attachments/assets/3ee9d337-2b9b-4f03-893c-cf7602a55365)

**Comment by lihuangbjtu** [comment]
Hi, I encountered the same issue. How did you resolve it?

**Comment by Gdnaiteab** [comment]
hello,I encountered the same issue when run testing code.How did you resolve it?

**Comment by Gadfly-Yang** [comment]
Hi, I encountered the same issue. How did you resolve it?
